package.json

```diff
@@ -52,12 +52,12 @@
     "eslint-config-okonet": "^5.0.1",
     "eslint-plugin-node": "^5.1.1",
     "husky": "^0.14.3",
-    "jest": "^20.0.4",
-    "jest-cli": "^20.0.4",
+    "jest": "^21.1.0",
+    "jest-cli": "^21.1.0",
     "jsonlint": "^1.6.2",
     "prettier": "1.5.3",
     "remove-lockfiles": "^1.1.1",
-    "strip-ansi": "^3.0.1"
+    "strip-ansi": "^4.0.0"
```

Comment:
Looks like `strip-ansi` is not being used in the code. @SimenB Could you please remove it from dev-dependencies?

Reply:
Is there a ESLint plugin for that? ;)

Reply:
done

Reply:
@okonet There seems to be something in the works - https://github.com/benmosher/eslint-plugin-import/issues/877. There's also [`depcheck`](https://github.com/depcheck/depcheck).